resources/compile-config.sh

```diff
@@ -5,19 +6,18 @@ cache_dir="$HOME/Library/Caches/com.runningwithcrayons.Alfred-2/Workflow Data/co
 mkdir -p "$cache_dir"
 
 # Get paths to configuration file
-installed_config="./config.applescript"
-cached_config="$cache_dir/config.applescript"
+installed_config="config.applescript"
 compiled_config="$cache_dir/config.scpt"
+cached_config_md5file="$cache_dir/config.applescript.md5"
 
-# Retrieve contents of config files
-installed_config_contents="$(cat "$installed_config" 2> /dev/null)"
-cached_config_contents="$(cat "$cached_config" 2> /dev/null)"
+# Retrieve md5sum of config files
+installed_config_md5sum=$(md5 "$installed_config" 2> /dev/null)
+cached_config_md5sum=$(< "$cached_config_md5file")
```

Comment:
Ah, I see. I just learned something new and interesting about `cat`. Thanks for that. :)